code/game/objects/effects/bump_teleporter.dm

```diff
@@ -19,19 +23,51 @@
 	LAZYREMOVE(AllTeleporters, src)
 	return ..()
 
-
 /obj/effect/bump_teleporter/singularity_act()
 	return
 
 /obj/effect/bump_teleporter/singularity_pull()
 	return
 
-/obj/effect/bump_teleporter/Bumped(atom/movable/AM)
-	if(!ismob(AM))
+/obj/effect/bump_teleporter/Bumped(atom/movable/concerned_party)
```

Comment:
This isn't obvious to me. I think `teleporting` or `bumper` would be better here.